### ISSUE TITLE
508-defect-2 [COGNITION]: The "Updated on Month ##, YYYY" after the page title SHOULD be consistent with the rest of VA.gov

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#13559]
#  [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Throughout VA.gov the "Updated on Month ##, YYYY" that appears beneath the page heading is presented in a `<p>` with regular paragraph styling. on the debt detail page, this information is presented in a `<p>` tag with custom styling that is inconsistent with the rest of the website. This introduces cognitive friction for our users. This information **should** be consistent to reduce cognitive friction and provide a trustworthy experience.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a user with a cognitive disability, I want to read VA.gov content that is styled consistently in order to confidently understand the hierarchy.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/manage-va-debt/your-debt/debt-detail in browser, signed in as user 1
2. Verify that the "Updated on Month ##, YYYY" that appears beneath the page heading is much large than the other paragraph text

### Current code

```
<p class="vads-u-font-size--h3 vads-u-font-family--serif vads-u-margin-top--0 vads-u-font-weight--normal">Updated on September 18, 2012</p>
```

### Potential fix

```
<p>Updated on September 18, 2012</p>
```

## WCAG or Vendor Guidance (optional)

* "The text SHOULD be written in the simplest and clearest language possible, appropriate to the content." [COG](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/WCAG-Checklist.md#part-4-understandabilitycognition)

## Screenshots or Trace Logs

![Screen Shot 2020-09-14 at 6 58 40 PM](https://user-images.githubusercontent.com/57469/93146264-521ae400-f6bc-11ea-83fd-81ada599b6b1.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by bradl3yC** [comment]
@jenstrickland can you give more context here? I just compared my code with the dates used in platform code and they're the exact same.  Also, this code was taken from the code you suggested in our 508 review of MDOT.

**Comment by rileyorr** [comment]
Hi, @jenstrickland! Thanks for bringing this to our attention :) 

I worry about the date becoming visually lost. Is a feasible solution for us to use `<p class = "va-introtext">` so it's a p tag and the styling aligns with our design?

**Comment by jenstrickland** [comment]
@rileyorr On the site the date is usually just a p tag, not the introtext styling.  Did you see it styled as introtext elsewhere?

**Comment by bradl3yC** [comment]
@jenstrickland in this instance, can a date not be included in introtext?

**Comment by jenstrickland** [comment]
We're supposed to have consistency across VA.gov. The users rely on it.

**Comment by bradl3yC** [comment]
@jenstrickland so no dates in intro text ... ever?

**Comment by rileyorr** [comment]
@jenstrickland I'm having a hard time following why we can't have dates in introtext styling. Users are familiar with seeing introtext styling under H1s, so it seems to align with consistency across the site.

**Comment by jenstrickland** [comment]
@rileyorr When I've seen the Updated on Month DD, YYYY on other pages it was in paragraph, not introtext, and other updated on date text is at the bottom of pages (the CMS pages, for example). The date would be similar to the "Equal to VA Form ####-##" language. The introtext styling is generally reserved for high-level overviews and introductions.  If you have an example of a page that uses introtext for this content, please let me know. 


**CMS pages**
![Screen Shot 2020-09-21 at 3 10 02 PM](https://user-images.githubusercontent.com/57469/93810375-8d696580-fc1c-11ea-9bd9-58b06fca1b06.png)

**Equal to VA Form 10-10EZ (Application for Health Benefits)**
![Screen Shot 2020-09-21 at 3 11 01 PM](https://user-images.githubusercontent.com/57469/93810560-d7524b80-fc1c-11ea-8af0-1e2ed662869c.png)

**Comment by rileyorr** [comment]
@jenstrickland thanks for providing those screenshots! I get where you're coming from, but I still disagree that the styling needs to match those other pages in this case. When our users navigate to the 'debt details' screen, they're looking for more information about their debt's status and whether or not they need to do anything on their end to progress towards debt resolution. A debt's resolution progression isn't linear, and oftentimes folks are waiting for documentation to be processed granting them a waiver, compromise offer, or extended payment plan. 

Navigating VA debt can be stressful, confusing, and terrifying for our users whose livelihoods rely on VA benefits. By making 'Updated on Month, DD, YYYY' more prominent, we're immediately and empathetically telling our users, "Hey, there's something new happening with your debt!" so they know there's more information available about the debt's status and potential next steps. Additionally, this type treatment matches the debt cards on the 'debt summary' screen, so I think it's important to carry that pattern throughout the tool (please see screenshot attached).  

<img width="675" alt="Screen Shot 2020-09-22 at 9 16 35 AM" src="https://user-images.githubusercontent.com/46329147/93887145-521a7580-fcb4-11ea-88f9-1ea894b58485.png">

I'm really glad we're having this discussion, though! It's important to provide context surrounding design decisions in our tools.

**Comment by jenstrickland** [comment]
Thanks for the background. Agreed, debt is stressful and providing clarity about the freshness of the data is vital. 

I'm not sure that providing it in introtext styling beneath the h1 serves that goal any more than using paragraph styling. I could imagine providing additional language with the date in introtext beneath the h1 working, though. It's good to see that the updated date is also provided next to the individual debts, which is where the users are most likely to want it.

**Comment by rileyorr** [comment]
Good morning, @jenstrickland! I brought this feedback to my team and we're in agreement that the date should remain visually prominent, so we're not going to change the date into paragraph styling at this time. We'll revisit this if through research we uncover that folks are experiencing cognitive difficulty understanding the page's hierarchy. Please let either me or @leahkeeler know if you have any questions!

**Comment by jenstrickland** [comment]
If research reveals that people with cognitive considerations don't understand the presentation, I wonder if it could be presented as a full sentence, rather than a fragment?
